res/values-es/strings.xml

```diff
@@ -5,27 +5,27 @@
     <string name="Apply">Aplicar</string>
     <string name="BangButtonExplanation">Buscar en otros lugares es fácil con !comandos</string>
     <string name="Cancel">Cancelar</string>
-    <string name="ChangingFontSize">Cambiando Tamaño de Letra&#8230;</string>
+    <string name="ChangingFontSize">Cambiando tamaño de letra&#8230;</string>
     <string name="Confirm">Confirmar</string>
-    <string name="ConfirmClearCacheAndCookies">El caché del navegador va a ser eliminado. ¿Estás seguro?</string>
```

Comment:
"cache" is usually used in feminine and passive tenses are not used very much in Spanish. So it would be much more correct: "La caché del navegador se va a eliminar""